### PR TITLE
Hdrp/staging pt6

### PR DESCRIPTION
**Pull request by eturquin**
Path tracing improvements:

- support for sphere lights (point lights with a radius)
- support of the cast_shadow flag for visibility computations
- improved (although still limited) handling of scene dirtiness, to reset iterations
- added transmittance color support on Lit surfaces
- added clearcoat support on the Lit surfaces
- added thin refraction support
- improved robustness of rough GGX computations
- fixed doc (no reference to tier 3 anymore)

(to merge after HDRP/staging_thin)

### CODE REVIEW

**Comment by anisunity** [comment]
@eturquin please fix the path tracing test and make sure it works. I can help if you do not know how to do it.

**Comment by eturquin** [comment]
Ok, will enable the test in the next PR.

**Comment by sebastienlagarde** [comment]
@remi-chapelain Please, be sure that Emissive intensity / SSR intensity is correct (compare to regualr SSR / emissive for example),and be sure path tracing itself isn't broken. Thanks

**Comment by eturquin** [comment]
All of Remy's points have been addressed:
- the point light issue is fixed
- shadows won't be culled at all (for the time being)
- the transmittance problem was a matter of changing few attributes of the shader

File: com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The current implementation for path tracing in the High Definition Render Pipeli
 
 ## Set up path tracing
 
-Path tracing shares the general requirements and setup as other ray tracing effects, so for information on hardware requirements and set up, see [getting started with ray tracing](Ray-Tracing-Getting-Started.html). The only difference is that you need to set the [Ray Tracing Tier](Ray-Tracing-Getting-Started.html#TierTable) to 3. You must carry out this setup before you can add path tracing to your Scene.
+Path tracing shares the general requirements and setup as other ray tracing effects, so for information on hardware requirements and set up, see [getting started with ray tracing](Ray-Tracing-Getting-Started.html). Note that you need to set the [Ray Tracing Tier](Ray-Tracing-Getting-Started.html#TierTable) to 2. You must carry out this setup before you can add path tracing to your Scene.
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md]: update

**Pull request merged**